data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/source/KafkaSource.java

```diff
@@ -296,7 +301,6 @@ private void setSchemaRegistryProperties(Properties properties, TopicConfig topi
         }
 
         if (schemaConfig.getType() == SchemaRegistryType.AWS_GLUE) {
```

Comment:
this can be removed? and have check only for confluent registry?

Reply:
It is the same thing whether to check for confluent and continue or check for glue and return. If a third registry were to come in future, this check is better.